app/models/complexity.rb

```diff
@@ -0,0 +1,10 @@
+class Complexity
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attribute :level, :string
+  attribute :reason, :string
+
+  validates_presence_of :level
+  validates_length_of :reason, minimum: 3, message: 'Enter the reason why the level has changed'
```

Comment:
The 3 character minimum still feels abitrary and out of context with the rest of the service, which simply has presence and a maximum of 175 as its boundaries for text boxes with reasons

Reply:
Eve has said she doesn't want a maximum but I will change minimum 3 to presence.

Reply:
In that case we need a definite comment to say that we are going against convention with our other text fields